angular-ui/src/app/app.module.ts

```diff
@@ -26,13 +27,16 @@ import { DatePipe } from '@angular/common';
     MovieListComponent,
```

Comment:
PasswordFormComponent needs to be in this declarations array

Reply:
Updated array